src/annotator/guest.js

```diff
@@ -208,6 +208,12 @@ export default class Guest {
       }
     });
 
+    addListener('mousedown', event => {
+      if (!annotationsAt(event.target).length) {
+        maybeCloseSidebar(event);
```

Comment:
Because we close this on mouse down, I think we can get rid of the `else` case line 207 in the `click` listener and any corresponding test. The new `mousedown` listener should now take care of this.

Reply:
You are correct, thanks.

Another change we could make it is to change the `click` event for `mouseup`. In that way, it would be symmetrical. What do you think?